src/tailwind.css

```diff
@@ -1,8 +1,186 @@
 /* purgecss start ignore */
+
 @tailwind base;
 @tailwind components;
 /* purgecss end ignore */
 
 /* Override utils here*/
 
 @tailwind utilities;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2530;
+}
+
+.joinUsWrapper {
```

Comment:
you can add the following classes to the wrapper in jsx right away `flex flex-col` , delete this CSS and selector